Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_20_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Defender for Endpoint
+
+Added new command **microsoft-atp-list-auth-permissions** to get the permissions of the currently configured user.
```

Comment:
Added the **microsoft-atp-list-auth-permissions** command to retrieve the permissions of the currently configured user.